docs/NSR.md

```diff
@@ -445,37 +413,69 @@ The CA SHOULD ensure incident response plans minimally include:
    2. containment of the incident to minimize further impact; and
    3. identification and mitigation or eradication of the incident root cause(s).
 
-### 4. Vulnerability Detection and Patch Management
+### 4. Vulnerability Management
+
+The CA MUST implement the policies and procedures in this Section for identifying, evaluating, and resolving security vulnerabilities.
+
+These policies and procedures MUST apply to all Certificate Systems.
+
+These policies and procedures SHOULD apply to Security Support Systems.
+
+#### 4.1 Inventory of Certificate Systems
+
+The CA MUST define an inventory of Certificate Systems.
```

Comment:
CS group feedback: what if we move this up so that it pertains not just to vuln management, which I don't believe was the intention?